app.js

```diff
@@ -4,16 +4,35 @@ const logger = require('morgan');
 const cookieParser = require('cookie-parser');
 const bodyParser = require('body-parser');
 const compression = require('compression');
-
 const session = require('express-session');
-
-// template stuff
+const methodOverride = require('method-override');
```

Comment:
is that used?

Reply:
I don't know. But you just changed the position of the import. I will keep it for now